pkgs/default.nix

```diff
@@ -116,4 +116,63 @@ rec {
 
   deploymentModules = self.deploymentModules;
 
-} // modules
+  allModules = [
+    (import ./moduleit/module-definition.nix)
+    # (import ./modules/bundles/go)
+    # (import ./modules/compilers/go)
+    # (import ./modules/languageServers/gopls)
+    # (import ./modules/formatters/gofmt)
+    # (import ./modules/bundles/ruby)
+    # (import ./modules/interpreters/ruby)
+    # (import ./modules/languageServers/solargraph)
+    # (import ./modules/packagers/rubygems)
+    # (import ./modules/bundles/nodejs)
+    # (import ./modules/interpreters/nodejs)
+    # (import ./modules/formatters/prettier)
+    # (import ./modules/languageServers/typescript-language-server)
+    # (import ./modules/packagers/nodejs-packager)
+    # (import ./modules/debuggers/js-debug)
+    # (import ./modules/bundles/bun)
+    # (import ./modules/interpreters/bun)
+    # (import ./modules/packagers/bun)
+    # (import ./modules/bundles/web)
+    # (import ./modules/languageServers/css-language-server)
+    # (import ./modules/languageServers/html-language-server)
```

Comment:
```suggestion
```

the dir structure of this repo has really been bothering me, let's fix that with this new era 😅

Reply:
What's the suggested change? I can't see the diff...